core-services/prow/02_config/containers/podman/_prowconfig.yaml

```diff
@@ -3,9 +3,11 @@ branch-protection:
     containers:
       repos:
         podman:
-          branches:
-            v1.6:
-              protect: false
+          protect: false  # Do not manage branch-protection settings.
+        required_status_checks:
+          contexts:
+            - success  # "Total Success" Cirrus-CI task must pass.
```

Comment:
I've been impossibly confused the last few days when reading comments like "branch protection goes by NAME", because there are many names. If the following comment is true (and it might not be, because I'm still confused), I think it might be more helpful for future maintainers:
```
 # Require the cirrus task with this ALIAS to pass
```

Reply:
There's different Human-names (for display) and machine-names (for use/reference).  I get your point though.  And this reminds me, I need to use the github API to actually check what prow/tide will see...

Reply:
...glad I checked, looks like it's using the human name.  For Example podman PR 21657, I find the check-run ID is directly in the 'details' url, so 21579329450:

curl -L -H "Accept: application/vnd.github+json" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/containers/podman/check-runs/21579329450

mentions:

`"name": "Total Success"`

So that's probably what needs to be referenced...force-push incoming...